Text/Megaparsec/Stream.hs

```diff
@@ -244,12 +243,12 @@ instance Stream BL.ByteString where
     | BL.null s = Nothing
     | otherwise = Just (BL.splitAt (fromIntegral n) s)
   takeWhile_ = BL.span
-  showTokens Proxy = stringPretty . fmap (chr . fromIntegral)
+  showTokens Proxy = prettyTokens . fmap (chr . fromIntegral)
   -- NOTE Do not eta-reduce these (breaks inlining)
   reachOffset o pst =
-    reachOffset' splitAtBL BL.foldl' BL8.unpack (chr . fromIntegral) (10, 9) o pst
+    reachOffset' splitAtBL BL.foldl' ((:[]) . chr . fromIntegral) ((==10), (==9)) o pst
   reachOffsetNoLine o pst =
-    reachOffsetNoLine' splitAtBL BL.foldl' (10, 9) o pst
+    reachOffsetNoLine' splitAtBL BL.foldl' ((==10), (==9)) o pst
```

Comment:
`Word8` could be made an instance of `SrcPosToken` and then `reachOffset[NoLine]` could avoid having to pass these `(==10)` arguments by using `isLF/isTab`.